docs/modules/help/pages/tickets_edit.adoc

```diff
@@ -0,0 +1,46 @@
+= Creating a ticket
+
+[[overview]]
+== Overview
+
+One of the most common use cases of the Control Panel consists in the creation of new tickets. You can access the ticket creation form from various parts of the user interface.
+
+WARNING: Please do not include secrets such as password or private keys in the text of the ticket; please refer to the xref:tickets_edit.adoc#secrets["Frequently Asked Questions" section below] for information about how to securely transmit such information.
+
+The animation below shows how to create a simple ticket, after clicking on the "Create new ticket" link in the xref:dashboard.adoc[Dashboard].
+
+.Creating a new ticket
+image::ticket-create.gif[]
+
+The steps for the creation of a new ticket are the following:
+
+IMPORTANT: Please enter all information in the ticket in English, as some VSHNeers don't speak German.
```

Comment:
IMHO this sounds too much like an order. If a customer isn't comfortable with writing a request in english, german is usually not a problem either. I'd prefer to leave away the "Important" and just write "We prefer tickets in English, as some VSHNeers don't speak German". I think that should be enough.